### PR TITLE
Dataset Upload

### PR DESCRIPTION
**Pull request by sanjaychelliah**
# what

- Adding ipynb notebooks
- Text upload from CSV
- Imagefiles, textfiles upload from folders
- Structuring the repo
- Readme updates

### CODE REVIEW

File: datasets/upload/README.md
Comment (sainivedh) [inline datasets/upload/README.md:1]: Is it possible to include an introduction regarding notebook in the README as well?

File: datasets/upload/dataset_upload.ipynb
Comment (sainivedh) [inline datasets/upload/dataset_upload.ipynb:1]: Minor Suggestions to reformat headings(upload_dataset, upload_from_folder ..etc) to just Upload Dataset and delete unused cells.

Comment (sainivedh) [reply datasets/upload/dataset_upload.ipynb:1]: Should we also show to create app with different base workflows when uploading different dataset types ?

**Review by sainivedh — COMMENTED** [review]
Looks good. Some Comments

File: datasets/upload/dataset_upload.ipynb
Comment (sanjaychelliah) [reply datasets/upload/dataset_upload.ipynb:1]: We can add a `basics.ipynb` notebook later to have create app, dataset etc...

File: datasets/upload/README.md
Comment (sanjaychelliah) [reply datasets/upload/README.md:1]: Its already in the main README with the link to [notebooks](https://github.com/Clarifai/examples/tree/DEVX-152-upload_text_files#notebooks).

**Review by sainivedh — APPROVED** [review]
Looks good

**Pull request merged**